AutoGraphTests/Stub.swift

```diff
@@ -1,10 +1,36 @@
 import Foundation
-import OHHTTPStubs
+
+fileprivate var AllStubs = [Stub]()
+
+extension Array where Element == Stub {
+    func find(_ request: URLRequest ) -> Data? {
+        for stub in self {
+            if stub.verify(request: request) {
+                return stub.responseData
+            }
+        }
+        return nil
+    }
+}
 
 class Stub {
+    static func clearAll() {
+        AllStubs.removeAll()
+    }
+    
     var json: Any? {
         if let jsonFixtureFile = self.jsonFixtureFile {
-            let path = Bundle(for: type(of: self)).path(forResource: jsonFixtureFile, ofType: "json")!
+            let path: String = {
+                #if os(iOS)
+                return Bundle(for: type(of: self)).path(forResource: jsonFixtureFile, ofType: "json")!
+                
+                #else
+                let fileManager = FileManager.default
+                let currentDirectoryPath = fileManager.currentDirectoryPath
+                return "\(currentDirectoryPath)/AutoGraphTests/Data/\(jsonFixtureFile).json"
+                
+                #endif
```

Comment:
iOS has a Bundle, but command line swift does not. This gets the right path for the stubbed data.